packages/to-json-schema/src/convertSchema.ts

```diff
@@ -167,263 +167,270 @@ export function convertSchema(
         jsonSchema = convertAction(jsonSchema, valibotPipeItem, config);
       }
     }
+  } else {
```

Comment:
Did you remove the `return` statement for a specific reason and replace it with `else`?

Reply:
I was able to fix the bug without this change and the new code at the end.